package.json

```diff
@@ -17,50 +16,42 @@
         "types": "./lib/core/index.d.mts",
         "default": "./lib/core/index.mjs"
       },
-      "react-native": {
-        "import": {
-          "types": "./lib/core/index.d.mts",
-          "default": "./lib/core/index.mjs"
-        },
-        "default": {
-          "types": "./lib/core/index.d.ts",
-          "default": "./lib/core/index.js"
-        }
-      },
       "import": {
         "types": "./lib/core/index.d.mts",
         "default": "./lib/core/index.mjs"
       },
-      "default": {
```

Comment:
Removing the `default` condition means that if `msw` import is resolved and none of the above conditions match, the import will fail as opposed to falling back to the CJS output. I believe this is a mistake.